### DISCUSSION TITLE
Is there a buffer source (like hrsh7th/cmp-buffer) that works for all open buffers?

### DISCUSSION THREAD

**Discussion by lehmacdj** [hrsh7th/nvim-cmp#856] [Q&A] [answered]
Is there a source like the hrsh7th/cmp-buffer that pulls keywords from all open buffers instead of just the current one?

This would be useful for situations where I was looking at the definition of a function and want to type it in a different file for example (and of course only in situations where I don't have a functioning LSP...).

**Comment by hrsh7th** [comment, marked as answer]
See this README.md https://github.com/hrsh7th/cmp-buffer#get_bufnrs-type-fun-number

**Comment by lehmacdj** [comment]
Thanks, this works great!
